docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: '2'
+services:
+  refocus:
+    image: "dbabenko/refocus"
```

Comment:
No. And I hope we can replace it with the reference to official public refocus docker image. 

This is a reference to my public docker repo https://hub.docker.com/r/dbabenko/refocus/
that I created to verify my changes. If the image exists then users don't need to build the image locally - one can just run "docker-compose up" and images for refocus, redis, postgres will be downloaded automatically. If there is no image  then user would have to build the image locally, modify docker-compose.yml to point to their own preferred repo. 

A couple of options to consider:

1. Refocus team can set up public docker repo and publish docker image there similar to how other projects release their images. 
2. Remove docker-compose.yml comletely and just provide instruction how one can create one. 

Maybe we should go with option 2 to start and see if there will be any interest in official public docker image. If you prefer option 2 I can create a new PR without any docker-compose.yml and move its content into docs section.